res_config_settings_enterprise_remove/README.rst

```diff
@@ -2,11 +2,11 @@
    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
    :alt: License: AGPL-3
 
-====================================
-Account - Remove Enterprise Features
-====================================
+===================================
+Settings - Remove Enterprise Fields
+===================================
 
-This module removes enterprise-only features from all account views.
+This module removes enterprise-only features from all settings views.
```

Comment:
I would add a note such as:
"This module does not remove the possibility to install the Enterprise modules: if necessary, all the code of Enterprise Edition is still available and can be manually installed via the Apps menu."